### PR TITLE
Add ppittle as AWS component co-owner

### PR DESCRIPTION
**Pull request by ppittle**
## Changes

- Add @ppittle as a co-owner for AWS components.

### CODE REVIEW

**Comment by ppittle** [comment]
Tagging @srprash and @birojnayak for review.

@Kielek - [Per your comment](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1539#pullrequestreview-1831623126) in #1539 - I have joined the OpenTelemetry organization :D

File: .github/component_owners.yml [hunk 1]
```diff
@@ -23,14 +23,17 @@ components:
...
     - atshaw43
+    - ppittle
   src/OpenTelemetry.Extensions.Enrichment/:
     - xakep139
   src/OpenTelemetry.Instrumentation.AWS/:
     - srprash
     - atshaw43
+    - ppittle
   src/OpenTelemetry.Instrumentation.AWSLambda/:
     - rypdal
```
Comment (utpilla) [inline .github/component_owners.yml:34]: @rypdal @Oberon00 Please review as you're the current owners of AWS Lambda instrumentation.

**Review by srprash — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

File: .github/component_owners.yml [hunk 2]
```diff
@@ -23,14 +23,17 @@ components:
   src/OpenTelemetry.Extensions.AWS/:
     - srprash
     - atshaw43
```
Comment (Kielek) [inline .github/component_owners.yml:25]: @ppittle, @srprash - side topic.

@atshaw43 is still not OTel member. He never accepted invitation to OpenTelemetry org. Could you please help with this or if he is no longer responsible for this package, he could self remove.

**Review by rypdal — APPROVED** [review]

**Review by Oberon00 — APPROVED** [review]
Great to have contributors from AWS on board!

**Pull request merged**